huaweicloud/services/acceptance/apig/resource_huaweicloud_apig_group_test.go

```diff
@@ -88,12 +156,29 @@ func testAccGroupImportStateFunc() resource.ImportStateIdFunc {
 
 func testAccGroup_base(name string) string {
 	return fmt.Sprintf(`
-%s
+variable "variables_configuration" {
+  type = list(object({
+    name  = string
+    value = string
+  }))
+  default = [
+    {name="TEST_VAR_1", value="TEST_VALUE_1"},
+    {name="TEST_VAR_2", value="TEST_VALUE_2"},
+    {name="TEST_VAR_3", value="TEST_VALUE_3"},
+    {name="TEST_VAR_2", value="TEST_VALUE_4"}, // same variable name, but value is different.
+  ]
+}
+
+%[1]s
 
 data "huaweicloud_availability_zones" "test" {}
 
+data "huaweicloud_apig_instances" "test" {
+  name = "tf_test_randx"
+}
```

Comment:
What is the significance of adding an unused datasource here?

Reply:
It is used for local test, I forgot to remove it.